src/test/java/seedu/address/model/person/OrganExpiryDateTest.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class OrganExpiryDateTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new OrganExpiryDate(null));
+    }
+
+    @Test
+    public void constructor_invalidOrganExpiryDate_throwsIllegalArgumentException() {
+        String invalidOrganExpiryDate = "";
+        assertThrows(IllegalArgumentException.class, () -> new OrganExpiryDate(invalidOrganExpiryDate));
+    }
+
+    @Test
+    public void isValidExpiryDate() {
+        // null organ's expiry date
+        assertThrows(NullPointerException.class, () -> OrganExpiryDate.isValidExpiryDate(null));
+
+        // invalid organ's expiry date
+        assertFalse(OrganExpiryDate.isValidExpiryDate("")); // empty string
+        assertFalse(OrganExpiryDate.isValidExpiryDate(" ")); // spaces only
+        assertFalse(OrganExpiryDate.isValidExpiryDate("^")); // only non-alphanumeric characters
+        assertFalse(OrganExpiryDate.isValidExpiryDate("23,Jan 2019")); // wrong data format
+        assertFalse(OrganExpiryDate.isValidExpiryDate("22 Jan 2019")); // wrong data format
```

Comment:
Could you add some checks for other date formats, eg DD/MM/YYYY, and checks for wrongly spelt months?